Sources/SwiftOracle/BindVar.swift

```diff
@@ -18,7 +18,7 @@ public class BindVar: StringLiteralConvertible, IntegerLiteralConvertible, Boole
         
     }
     public init (_ value: String) {
-        var v = Array(value.nulTerminatedUTF8).map( {Int8(bitPattern: $0) })
+        var v = Array(value.utf8CString)
```

Comment:
Compile Swift Module 'SwiftOracle' (5 sources)
/home/lip/for_orcl_driver/Packages/SwiftOracle-0.1.5/Sources/SwiftOracle/Row.swift:42:26: error: 'String' is not convertible to 'AnyObject'; did you mean to use 'as!' to force downcast?
            return "asd" as AnyObject
                   ~~~~~~^~~~~~~~~~~~
                         as!
/home/lip/for_orcl_driver/Packages/SwiftOracle-0.1.5/Sources/SwiftOracle/Connection.swift:114:19: warning: value 'connection' was defined but never used; consider replacing with boolean test
        guard let connection = connection else {
              ~~~~^~~~~~~~~~~~~
                                          != nil
<unknown>:0: error: build had 1 command failures
error: exit(1): /swift-3.0/usr/bin/swift-build-tool -f /home/lip/for_orcl_driver/.build/debug.yaml